project/accounts/models.py

```diff
@@ -9,3 +9,135 @@ class User(AbstractUser):
     """
     class Meta:
         db_table = 'users'
+
+class Profile(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=63, blank=False)
+    last_name = models.CharField(max_length=63, blank=False)
+    about_me = models.CharField(max_length=511, blank=True)
+
+    categories = models.ManyToManyField(
+        Category, related_name="user_categories", symmetrical=False
+    )
+    tags = TaggableManager()
+
+    followers = models.ManyToManyField(
+        "self", related_name="follower", symmetrical=False
+    )
+    following = models.ManyToManyField(
+        "self", related_name="followings", symmetrical=False
+    )
+
+    beta_access = models.BooleanField(default=False)
+    is_verified = models.BooleanField(default=False)
+    full_account = models.BooleanField(default=False)
+
+    objects = AccountManager()
+    profile_image = models.ImageField(
+        upload_to=profile_upload_path, blank=True, null=True
+    )
+    profile_image_thumb = models.ImageField(
+        upload_to=profile_upload_path, blank=True, null=True
+    )
+
+    @property
+    def full_name(self):
+        "Returns the person's full name."
+
+        full_name = "{first_name} {last_name}".format(
+            first_name=self.first_name, last_name=self.last_name
+        )
```

Comment:
Let's use f-strings here

```suggestion
        full_name = f"{ self.first_name } { self.last_name }"
```